suite-native/alerts/src/components/AlertSheet.tsx

```diff
@@ -87,12 +95,25 @@ export const AlertSheet = ({ alert }: AlertSheetProps) => {
                     >
                         <Card style={applyStyle(alertSheetContainerStyle)}>
                             <VStack style={applyStyle(alertSheetContentStyle)} spacing="large">
-                                <Pictogram
-                                    title={title}
-                                    variant={pictogramVariant}
-                                    subtitle={description}
-                                    icon={icon}
-                                />
+                                {icon && pictogramVariant ? (
+                                    <Pictogram
+                                        title={title}
+                                        variant={pictogramVariant}
+                                        subtitle={description}
+                                        icon={icon}
+                                    />
+                                ) : (
+                                    <VStack alignItems="center">
+                                        <Box>
+                                            <Text variant="titleSmall" textAlign="center">
+                                                {title}
+                                            </Text>
+                                        </Box>
+                                        <Text color="textSubdued" textAlign="center">
+                                            {description}
+                                        </Text>
+                                    </VStack>
+                                )}
```

Comment:
This composition looks weird, but I can see how you got there.

Cleaner composition could be done by extracting two components out of Pictogram - something like RingIcon and CenteredHeading, use those two new components in "Pictogram" and use them here without Pictogram. What do you think?

BTW, Pictogram is also confusing name for me, I wouldn't expect it to include a title and subtitle.

Reply:
Makes sense. I’ll refactor it to Pictogram (circle and icon) and PictogramHeader (Pictogram + CeneteredTitleHeader)

Reply:
ec77327397